lib/spack/env/cc

```diff
@@ -320,7 +320,7 @@ esac
 #
 # Unset pesky environment variables that could affect build sanity.
 #
-unset LD_LIBRARY_PATH
+#unset LD_LIBRARY_PATH
```

Comment:
Sorry this was not a "real change" for the pull request, just some intermediate mucking around.